dali/operators/image/convolution/gaussian_blur.cc

```diff
@@ -214,6 +214,7 @@ class GaussianBlurOpCpu : public OpImplBase<CPUBackend> {
   void RunImpl(workspace_t<CPUBackend>& ws) override {
     const auto& input = ws.template InputRef<CPUBackend>(0);
     auto& output = ws.template OutputRef<CPUBackend>(0);
+    output.SetLayout(input.GetLayout());
```

Comment:
```suggestion
    output.SetLayout(InputLayout(ws, 0));
```

Reply:
Hmm, there are not default layouts in schema, and I can support anything with up to 3 data axes + 'F' and 'C' somewhere around it.

Reply:
I won't push it then for the sake of consistency/uniformity.